go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/go-kit/kit v0.9.0
 	github.com/google/uuid v1.1.1
 	github.com/lestrrat-go/jwx v0.9.0
-	github.com/optimizely/go-sdk v1.5.1
+	github.com/optimizely/go-sdk v1.5.2-0.20201229233509-7c2b19099252
```

Comment:
This should not be merged/released until we have a full release (1.5.2) of Go SDK that can be used.